x-pack/test/alerting_api_integration/spaces_only/tests/alerting/migrations.ts

```diff
@@ -443,5 +443,22 @@ export default function createGetTests({ getService }: FtrProviderContext) {
       expect(response.statusCode).to.equal(200);
       expect(response.body._source?.alert?.tags).to.eql(['test-tag-1', 'foo-tag']);
     });
+
+    it('8.3.0 removes snoozeEndTime in favor of snoozeSchedule', async () => {
```

Comment:
Yes, this is because the migration will succeed even if decryption fails. I left a comment [that we should add an expect here to ensure the api key still exists after migration](https://github.com/elastic/kibana/pull/135602/files#r911469196) but that can't be done until [this](https://github.com/elastic/kibana/pull/135602/files#r911470061) (loading the test data in the correct manner)

Reply:
Brilliant, thanks, I'll work with Mike to get that sorted asap.